packages/react/src/components/Tabs/Tabs.js

```diff
@@ -282,42 +337,58 @@ export default class Tabs extends React.Component {
       );
     });
 
+    const leftOverflowNavButtonHidden =
+      !this.state.horizontalOverflow || !this.state.tablistScrollLeft;
+    const rightOverflowNavButtonHidden =
+      !this.state.horizontalOverflow ||
+      this.state.tablistScrollLeft + this.state.tablistClientWidth ===
+        this.state.tablistScrollWidth;
     const classes = {
       tabs: classNames(`${prefix}--tabs`, className, {
         [`${prefix}--tabs--container`]: type === 'container',
+        [`${prefix}--tabs--light`]: light,
+      }),
+      tablist: classNames(`${prefix}--tabs__nav`),
+      leftOverflowButtonClasses: classNames({
+        [`${prefix}--tab--overflow-nav-button`]: this.state.horizontalOverflow,
+        [`${prefix}--tab--overflow-nav-button--hidden`]: leftOverflowNavButtonHidden,
       }),
-      tablist: classNames(`${prefix}--tabs__nav`, {
-        [`${prefix}--tabs__nav--hidden`]: this.state.dropdownHidden,
+      rightOverflowButtonClasses: classNames({
+        [`${prefix}--tab--overflow-nav-button`]: this.state.horizontalOverflow,
+        [`${prefix}--tab--overflow-nav-button--hidden`]: rightOverflowNavButtonHidden,
       }),
     };
 
-    const selectedTab = this.getTabAt(this.state.selected, true);
-    const selectedLabel = selectedTab ? selectedTab.props.label : '';
-
     return (
       <>
-        <div {...other} className={classes.tabs} role={role}>
-          <div
-            role="listbox"
-            aria-label={ariaLabel}
-            tabIndex={0}
-            className={`${prefix}--tabs-trigger`}
-            onClick={this.handleDropdownClick}
-            onKeyPress={this.handleDropdownClick}>
-            <a
-              tabIndex={-1}
-              className={`${prefix}--tabs-trigger-text`}
-              href={triggerHref}
-              onClick={this.handleDropdownClick}>
-              {selectedLabel}
-            </a>
-            <ChevronDown16 aria-hidden="true">
-              {iconDescription && <title>{iconDescription}</title>}
-            </ChevronDown16>
-          </div>
-          <ul role="tablist" className={classes.tablist}>
+        <div
+          {...other}
+          className={classes.tabs}
+          role={role}
```

Comment:
this is carried over from the existing implementation (https://github.com/carbon-design-system/carbon-components-react/pull/2318). what is the recommendation here?

Reply:
Just took a look at our [design guidance for tabs](https://www.carbondesignsystem.com/components/tabs/usage#variations)

If the tabs are 'default' then the nav role makes sense actually. If it's a 'container' version then we can follow the [APG guidance](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html) and leave it out. If that's a pain in the neck having the nav role in there isn't a blocker. Since it reads and operates fine, just a little wonky and verbose in JAWS.

Reply:
Yes, please take out the nav for the 'container' Tabs. See: https://github.com/carbon-design-system/carbon/issues/2272

I'm not quite sure what 'default' Tabs are, but typically you would want to leave landmark design, like adding a nav, main, etc, up to the page author. (i.e. I think the nav should probably be taken out of 'default' Tabs, too).